src/main/java/frc/robot/subsystems/PhotonSubsystem.java

```diff
@@ -117,12 +120,13 @@ public Command getWaitForDataCommand(int tagid){
  * @return
  * the command to run
  */
-  public Command getAprilTagCommand(PhotonPositions spacePositions){
+  public Command getAprilTagCommand(PhotonPositions spacePositions, CommandXboxController driverStick){
     Command swerveRequirementCommand = Commands.run(() -> {}, SwerveSubsystem.getInstance())
             .withName("AprilTagCommandSwerveRequirement");
     if (spacePositions.hasWaypoint){
       return Commands.sequence(
         getWaitForDataCommand(spacePositions.id),
+        new RumbleJoystick(driverStick, RumbleType.kBothRumble, 0.8, 0.2, false),
```

Comment:
`new Schedule(new RumbleJoystick(driverStick, RumbleType.kBothRumble, 0.8, 0.2, false))`

Reply:
Add for both sides of the if statement (waypoint and no waypoint)

Reply:
change made